django/contrib/gis/gdal/prototypes/geom.py

```diff
@@ -137,7 +137,7 @@ def topology_func(f):
 get_geom_type = int_output(lgdal.OGR_G_GetGeometryType, [c_void_p])
 get_point_count = int_output(lgdal.OGR_G_GetPointCount, [c_void_p])
 get_point = void_output(
-    lgdal.OGR_G_GetPoint,
+    lgdal.OGR_G_GetPointZM,
     [c_void_p, c_int, POINTER(c_double), POINTER(c_double), POINTER(c_double)],
```

Comment:
I think this should be updated?

```suggestion
    [
        c_void_p, c_int, POINTER(c_double), POINTER(c_double), POINTER(c_double),
        POINTER(c_double),
    ],
```

See the [documentation](https://gdal.org/api/vector_c_api.html#_CPPv416OGR_G_GetPointZM12OGRGeometryHiPdPdPdPd).

Reply:
Yes. Quite.

Need to work out how/why the tests passed.

Reply:
I'm not sure why the the tests passed with an incorrect signature. While the python [docs](https://docs.python.org/3/library/ctypes.html#ctypes._FuncPtr.argtypes) have the following comment, I'm not entirely sure if that is the scenario here.

> functions using the C calling convention accept additional, unspecified arguments as well.

If we're happy the signature is now correct and the tests pass, is that enough?

Reply:
I can live with that.